src/components/Chip/stories.scss

```diff
@@ -0,0 +1,21 @@
+.story--components-buttontw--default {
+  background-color: red;
+}
+
+.story-wrapper {
```

Comment:
also we shd follow a strict rule of casing. kebap-case and camelCase etc are used

Reply:
Done